tests/hash-playwright/tests/signup.spec.ts

```diff
@@ -27,15 +27,15 @@ test("user can sign up", async ({ page }) => {
 
   await page.fill('[type="password"]', "some-complex-pw-1ab2");
 
-  await page.click("text=Sign up with email");
+  await page.click("text=Sign up");
 
   await expect(
     page.locator("text=Thanks for confirming your account"),
   ).toBeVisible();
 
-  await page.fill('[placeholder="example"]', randomNumber.toString());
+  await page.fill('[id="shortname"]', randomNumber.toString());
 
-  await page.fill('[placeholder="Bobby"]', "New User");
+  await page.fill('[id="name"]', "New User");
```

Comment:
I can do this, but don't quite understand the thinking since we are essentially then saying in the tests "There must be an input with placeholder "Jonathan Smith"", rather than just "There must be a "name" input which can be filled". If we change the placeholder again, the tests would break again whereas if we stick with the ID the tests would break if we change the inputs entirely.

Happy to address this though, don't feel very strongly. But that was my thinking.

Reply:
The thinking is to keep the interaction Playwright is running as close to something a real user sees when interacting with the app, because then we're testing what a user experiences rather than a machine... they can see the palceholder, they can't see the id of the input. Even better would be clicking a label which then focused the input, but the placeholder is a bit better.

Reply:
I can see the argument for wanting to test whether the placeholder is visible over the id, have addressed as part of https://github.com/hashintel/hash/pull/4009/commits/af3f0133e1e225b60b23788cf874c0f6088a4d69